bindings/python/build-libbson.sh

```diff
@@ -22,14 +22,17 @@ fi
 # Directory where build artifacts will be placed
 LIBBSON_INSTALL_DIR=${LIBBSON_INSTALL_DIR:-""}
 
+# Replace a relative path with an absolute on for cmake
+LIBBSON_INSTALL_DIR=${LIBBSON_INSTALL_DIR/.\//$(pwd)\/}
```

Comment:
I suggest using cd+pwd+basename to handle relative paths more generally:
```
LIBBSON_INSTALL_DIR="$(cd "$(dirname "$LIBBSON_INSTALL_DIR")"; pwd)/$(basename "$LIBBSON_INSTALL_DIR")"
```

Reply:
Done